### DISCUSSION TITLE
Student verification - school name issue

### DISCUSSION THREAD

**Discussion by Samuel-Pr** [community/community#67383] [GitHub Education]
### Hi everyone,

I have a problem when uploading "proof of your academic status". I am uploading my ISIC card and when I submit it, it throws error:
Your application cannot be reviewed until you fix the following:

    Please make sure the document you uploaded includes the name of the school which you selected.
    The image you selected does not appear to contain your school name. Ensure your image (photo or upload) is legible. We do not accept school logos. If you are taking a photo of a screen try to have it as large a resolution as possible. You may include multiple documents in your image. If your original document is not in English then you might take a photo of the original next to an English translation.

Name of school is correct. I don't know if there is problem with that the github have our school as SPŠE (short form) K. Adlera, when on ISIC have long form of school name or there is problem with special characters as š, á, č.

Thanks for your help.

**Comment by elliemgmc** [comment]
It might be that the image is too blurry and cannot be read. Try making the resolution better or put multiple documents.
